langs/j/Dockerfile

```diff
@@ -1,35 +1,29 @@
-ARG VER=9.6
-
-FROM debian:bookworm-slim AS builder
-
-ARG VER
+FROM debian:trixie-slim AS builder
 
 RUN apt-get update                   \
  && DEBIAN_FRONTEND='noninteractive' \
-    apt-get install --yes curl
+    apt-get install --yes clang git make nasm
+
+ENV CC='clang' CFLAGS='-Wno-error' VER=9.6
+
+RUN git clone --branch J$VER-beta3 --progress \
```

Comment:
Oh this doesn't look right, we were on 9.6.3 before, not 9.6-beta3. Looks like this downgrade caused some issues - https://discord.com/channels/756829356155731988/758369341460054038/1425364247264628858